tools/DependencyUpdater/Program.cs

```diff
@@ -31,8 +31,17 @@ static int Main(string[] args)
                 else if (File.Exists(arg))
                 {
                     string sem = File.ReadAllText(arg);
-                    string share = sem.Substring(sem.LastIndexOf(':') + 1).Trim();
-                    sources.Add(Path.Combine(share, @"Signed\Packages") + (NoTimestamp ? "-NoTimeStamp" : ""));
+
+                    foreach (string line in sem.Split('\n'))
+                    {
+                        string share = line.Trim().Substring(line.LastIndexOf(':') + 1).Trim();
```

Comment:
I think you need to get rid of the first Trim(), otherwise the substring index could be off.
`line.LastIndexOf(':')`
operates on the untrimmed version of line. But the substring operates on 
`line.Trim()`

One case which will not be correctly processed is a line with leading space, e.g ` blah :foo ` (with a leading space before blah).